app/src/main/res/layout/time_dialog.xml

```diff
@@ -64,6 +65,46 @@
                 android:layout_height="wrap_content"
                 android:id="@+id/disableSleeptimerButton"/>
 
+        <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="0dp"
+                android:layout_weight="1"
```

Comment:
I think it should be enough to just use:

```suggestion
                android:layout_height="wrap_content"
```

and then use `android:layout_weight` in the children (like you already do). You can use `1` for each child - no need to end up with a total of `1`.